File: app/src/main/java/in/woloo/www/giftcard/mvp/GiftCardView.java
package in.woloo.www.giftcard.mvp;

import java.io.Serializable;

import in.woloo.www.giftcard.model.RequestPointsResponse;
import in.woloo.www.more.models.UserCoinsResponse;

public interface GiftCardView extends Serializable {
    void showResult(String message);
    void userCoinsResponseSuccess(UserCoinsResponse userCoinsResponse);
    void RequestPointsResponseSuccess(RequestPointsResponse requestPointsResponse);
    void pointsAddedResponseSuccess();
}

File: app/src/main/java/in/woloo/www/more/giftcard/mvp/GiftCardView.kt
package `in`.woloo.www.more.giftcard.mvp

import `in`.woloo.www.more.giftcard.model.RequestPointsResponse
import `in`.woloo.www.more.models.UserCoinsResponse
import java.io.Serializable

interface GiftCardView : Serializable {
    fun showResult(message: String?)
    fun userCoinsResponseSuccess(userCoinsResponse: UserCoinsResponse?)
    fun RequestPointsResponseSuccess(requestPointsResponse: RequestPointsResponse?)
    fun pointsAddedResponseSuccess()
}
